src/docs/release-notes/R21/R21.mdx

```diff
@@ -0,0 +1,47 @@
+**Important**: we will be renamed to _Brightlayer User Interface_ (BLUI) soon, but our rapport to you all will stay the same 🤗
+
+### Component Libraries
+
+-   Updated Angular library to build with Angular 12.
+-   Documented all the components with JSDocs. (This is the intellisense thing)
+-   Enhanced React Native component library to allow icons passed directly into components.
+-   All PX Blue React Native component now responds to device rotation properly.
```

Comment:
```suggestion
-   All PX Blue React Native components now responds to device rotation properly.
```

Reply:
Is this new or did we do this last release?

Reply:
The story is in this release.